overview.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: info
```

Comment:
Note to self: Didn't test this yet.

Reply:
On md basis there is not enough jekyll foo available. because of this the extension of the page layout was made and dubbed info. The code for this layout is seen in _layouts/info.html